Makefile

```
@@ -160,6 +160,7 @@ clean:
 	find . -name '*.created' -exec rm -f {} +
 	find . -name '*.pyc' -exec rm -f {} +
 	rm -rf certs *.tar vendor $(NODE_CONTAINER_BIN_DIR)
+	rm dist/calicoctl
```

```suggestion
	rm -f dist/calicoctl
```

without `-f`, the exit code of rm != 0 if the file is missing.

Should the whole `dist` directory be deleted?

---

- or -f ... ack

I would guess dist/*, waiting for opinion from @caseydavenport & co.

---

Yeah I think we can blow away all of `dist` in `clean`

---

ack, done